{
  try {
    connector.close();
  }
 catch (  Exception e) {
  }
}
